internal/agents/pi/model_routing_client.go

```
@@ -0,0 +1,142 @@
+	Ok       bool   `json:"ok"`
+}
+type InspectRequest struct{ Path string `json:"path"` }
+type InspectResponse struct{ Contract string `json:"contract"` }
+type ValidateResponse struct{ Contract string `json:"contract"` }
+type ApplyResponse struct{ Contract string `json:"contract"` }
+
+func (c *ModelRoutingClient) do(ctx context.Context, op string, payload json.RawMessage) ([]byte, error) {
+	if c.Bin == "" {
+		return nil, &RoutingError{Kind: "missing", Path: c.Bin}
+	}
+	data, _ := json.Marshal(wireRequest{Contract: modelRoutingContract, Op: op, Payload: payload})
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `encoding/json.RawMessage` found

(errchkjson)